UI/MapWnd.cpp

```diff
@@ -5779,16 +5779,36 @@ void MapWnd::RefreshFleetResourceIndicator() {
     const std::set<int>& this_client_known_destroyed_objects = GetUniverse().EmpireKnownDestroyedObjectIDs(empire_id);
 
     int total_fleet_count = 0;
```

Comment:
if adding other counts of ships (not actually counting fleets), this should be renamed to total_ship_count